src/public/tests/questions_router_test.rs

```diff
@@ -15,7 +15,10 @@ mod tests {
         in_memory::question::QuestionInMemoryRepository,
         postgres::question_db::{QuestionDBRepository, MIGRATIONS},
     };
-    use cli::router::Router;
+    use cli::{
```

Comment:
`cargo doc` is failing because of this line

Reply:
![Screenshot 2024-03-27 at 18 05 52](https://github.com/sonntuet1997/rust-web-api-microservice-template/assets/33181397/0711b669-627c-4a15-af0e-b23ce0d03a0b)